content/docs/command-reference/remove.md

```diff
@@ -9,7 +9,7 @@ optionally delete them).
 usage: dvc remove [-h] [-q | -v] [--outs] targets [targets ...]
 
 positional arguments:
-  targets        stages (found in dvc.yaml) or .dvc files to remove.
+  targets        stages (found in dvc.yaml), .dvc files or name of output files to remove.
```

Comment:
let's please not use the `output` term here . `output` is considered more of an internal DVC terminology, for end-users it's better to use data files, model files, directories - that kind of terminology. Usually,  something that anyone would immediately understand is better than a technical term

Reply:
It feels the Description below, needs some update to it as well

especially part like this:

> `dvc remove` doesn't remove files from the DVC <abbr>cache</abbr> or
> [remote storage](/doc/command-reference/remote). Use `dvc gc` for that.

do we change this behavior? what is the most common scenario we now expect? etc

cc @dberenbaum

Reply:
> let's please not use the `output` term here . `output` is considered more of an internal DVC terminology, for end-users it's better to use data files, model files, directories - that kind of terminology. Usually, something that anyone would immediately understand is better than a technical term

@shcheklein I understand your point and I have been trying to find a better name, without much success. Technical writing in English is not my strength :sweat_smile: 

In this case, I feel that we need to use the `output` term here, in the same way it is being used in other places along the doc page, because we are actually talking about internal DVC terminology (i.e. we are referring to the `outpus` of a `stage`). 

For sure the semantics surrounding the `output` word can be improved and I will try to do that.

> It feels the Description below, needs some update to it as well
> 
> especially part like this:
> 
> > `dvc remove` doesn't remove files from the DVC cache or
> > [remote storage](/doc/command-reference/remote). Use `dvc gc` for that.
> 
> do we change this behavior? what is the most common scenario we now expect? etc
> 
> cc @dberenbaum

That behavior did not changed. I guess the confusion is my fault for not updating other parts of the doc page. The change introduced in the DVC P.R. changes the behavior described a little bellow that section:

```
Note that the actual output files or directories of the stage (outs field) are not removed by this command, unless the --outs option is used.
```

After the change in the DVC P.R. (and what should we properly document here), you can now remove *concrete* (i.e. one of many) output files or directories of the stage (outs field). Previously you could only remove *all* the outputs as described in that section.

Reply:
> for end-users it's better to use data files, model files, directories - that kind of terminology

The problem with that may be that we specifically mean stage outputs, not dependencies (which can also be all of those things) — same as https://github.com/iterative/dvc/pull/5273#discussion_r609853710.

Reply:
> we specifically mean stage outputs here, not dependencies

Actually I guess indeed "outputs" doesn't make that much sense in the context of add/remove.

> we are referring to the outpus of a stage

Not necessarily, @daavoo. .dvc files aren't stages and while they have the `outs` field inside that's a just the internal terminology like Ivan mentioned. How about:

```suggestion
  targets        Tracked files/directories, stage names (found in
                 dvc.yaml), or .dvc files to remove.
```

Reply:
> The change introduced in the DVC P.R. changes the behavior described a little bellow that section:
> `Note that the actual output files or directories of the stage (outs field) are not removed...`

@daavoo how is that changed by iterative/dvc/pull/5757? In the case of stage name targets. Thanks

Reply:
My bad for not clarifying. There are no changes in the case of stage name targets.

I was referring to the last part of that statement `unless the --outs option is used.` and trying to say that `--outs` is no longer the only option to remove `the actual output files or directories of the stage`

Reply:
Gotcha, sounds like we're on the same page now then. Thanks!